sycl/include/sycl/ext/oneapi/experimental/syclbin_properties.hpp

```diff
@@ -0,0 +1,39 @@
+//==-------- syclbin_properties.hpp - SYCLBIN and tooling properties -------==//
```

Comment:
AFAIK, this part of the style guide has been dropped (having file name present in the very first line).

Reply:
Do you mean it should be
```suggestion
//==-------------------- SYCLBIN and tooling properties --------------------==//
```
or should the header be dropped entirely?